examples/llm/scripts/export_for_inference.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 MosaicML Examples authors
```

Comment:
Could you add a description for this script (similar to the PR description) in the header here?

Reply:
Done